NGitLab.Mock/Clients/IssueClient.cs

```diff
@@ -302,6 +302,14 @@ public async Task<Models.Issue> GetByIdAsync(int issueId, CancellationToken canc
             return GetById(issueId);
         }
 
+        public IEnumerable<Models.Issue> LinkedTo(int projectId, int issueId)
+        {
+            using (Context.BeginOperationScope())
+            {
+                return Enumerable.Empty<Models.Issue>();
```

Comment:
If you can't implement it, you should make it clear the method is not implemented:

```suggestion
                throw new NotImplementedException();
```

Reply:
Sure, I'll take care of all those raised issues